cmd/node/main.go

```diff
@@ -604,6 +610,44 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 	return nil
 }
 
+func startStatusPolling(ash core.AppStatusHandler, pollingInterval int, networkComponents *factory.Network) error {
+	if ash != nil {
+		pollingInterval := time.Duration(pollingInterval) * time.Second
+		appStatusPollingHandler, err := appStatusPolling.NewAppStatusPolling(ash, pollingInterval)
+
+		if err != nil {
+			return errors.New("cannot init AppStatusPolling")
+		} else {
```

Comment:
Useless else, you returned on first condition. In most cases is recommended to avoid else conditions because they can create spaghetti code. Simple if's are more readable.

Reply:
done